pom.xml

```diff
@@ -1006,6 +1008,33 @@
         <artifactId>nuxeo-platform-pdf-utils</artifactId>
         <version>${nuxeo.pdfutils.version}</version>
       </dependency>
+
+      <!-- 3D -->
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-api</artifactId>
+        <version>${nuxeo.threed.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-core</artifactId>
+        <version>${nuxeo.threed.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-convert</artifactId>
```

Comment:
That is a great question. 
Did it to follow video and imaging patterns. Thought that it would be easier to have "similar" things with the same structure to avoid more complexity.
But it is a good candidate for a future refactor (on all projects), to figure out the best approach.